packages/builders/src/interactions/slashCommands/SlashCommandBuilder.ts

```diff
@@ -44,7 +58,9 @@ export class SlashCommandBuilder {
 
 		return {
 			name: this.name,
+			name_localizations: this.name_localizations,
```

Comment:
Oh dear.. this doesn't do the same spreading pattern as options do... I'll fix that after this PR